test/e2e/framework/util.go

```diff
@@ -2230,6 +2230,20 @@ func ListSchedulableNodesOrDie(c *client.Client) *api.NodeList {
 	return nodes
 }
 
+func ListReadyNodes(f *Framework) (nodes *api.NodeList, err error) {
```

Comment:
the Unschedulable part was in networking but now looks like it found its way into utils i guess, so no longer needed.